src/bundle/Resources/views/themes/admin/ui/dashboard/tab/my_content.html.twig

```diff
@@ -46,4 +46,5 @@
     ],
     body_rows,
     empty_table_info_text: 'dashboard.tab.my_content.empty'|trans|desc('No content. Content items you create will show up here'),
+    is_last_column_sticky: false,
```

Comment:
This will make the last column not sticky, which is a feature to show actions when the table has a big width. 
This should be done in a different way. To keep a sticky column and show the widget properly.